composer.json

```diff
@@ -27,13 +27,13 @@
   },
   "require": {
     "php": ">=5.4.8",
-    "phly/http": "^0.12",
-    "psr/http-message": "^0.10",
+    "psr/http-message-implementation": "~1.0.0",
```

Comment:
I have the package, but when there is no `dev` I am getting an error.

```
Your requirements could not be resolved to an installable set of packages.

  Problem 1
    - The requested package psr/http-message-implementation could not be found in any version, there may be a typo in the package name.

Potential causes:
 - A typo in the package name
 - The package is not available in a stable-enough version according to your minimum-stability setting
   see <https://groups.google.com/d/topic/composer-dev/_g3ASeIFlrc/discussion> for more details.

Read <https://getcomposer.org/doc/articles/troubleshooting.md> for further common problems.
```

But when I add `"phly/http": "~0.13"` to the `require-dev` all goes well. Do you have any idea how this will go when we don't add the dependency on require ? May be composer should automatically suggest the virtual package.